DependencyInjection/JKMoneyExtension.php

```diff
@@ -21,7 +21,7 @@ class JKMoneyExtension extends Extension implements PrependExtensionInterface
     public function load(array $configs, ContainerBuilder $container)
     {
         $locale = $container->getParameter('kernel.default_locale');
```

Comment:
I think this line needs to be removed entirely - that was the entire issue right? That this line returned something else then expected. The locale for twig needs to be resolved in some other way. Not sure how though, I don't write extensions that often.

Reply:
So, what I did recently was this:

```
services:
    Money\Formatter\IntlMoneyFormatter:
        $formatter: '@App\Money\CurrencyNumberFormatter'
```

```
<?php

declare(strict_types=1);

namespace App\Money;

class CurrencyNumberFormatter extends \NumberFormatter
{
    public function __construct(string $locale)
    {
        parent::__construct($locale, self::CURRENCY);
    }
}
```

Where `$locale` is autowired and then in Twig extension:

```
    public function getFilters(): array
    {
        return [
            new TwigFilter('money', [$this->intlMoneyFormatter, 'format']),
        ];
    }
```

I'm not sure if this would work for that bundle. As I said, I don't use it any more.

Reply:
Guess the PR will die here then. Basicaly we need to instruct MoneyExtension to use `%kernel.default_locale%` parameter value but without loading it here since it's not yet resolved. I don't know the correct syntax though.

Reply:
Okay I tried it and it seems it's actually very simple. Just remove this line and use this for the TwigExtension down below:

```
$twigExtension->replaceArgument(0, '%kernel.default_locale%');
```